fbench/src/httpclient/httpclient.cpp

```diff
@@ -510,7 +522,7 @@ HTTPClient::Close()
             || _connectionCloseGiven
             || !_dataDone
             || (_httpVersion == 0 && !_keepAliveGiven)) ?
-        _socket->Close() : true;
+        (_socket.reset(), true) : true;
```

Comment:
Comma operator caught me a bit off guard..! 🙂 Could consider factoring this out to a separate `if` for the `_socket.reset()` since this function seems to always return `true` anyway?

Reply:
I usually do not use the comma operator. I was just going for the 'minimal change for equivalent behavior' pattern to avoid triggering the 'break random stuff by refactoring at the same time' pattern. In other words; I agree that the code is not optimal, but it is easier to visually verify that the change does not change behavior compared to a more refactored approach.